.eslintrc.cjs

```diff
@@ -28,5 +28,12 @@ module.exports = {
         patterns: [{ group: ['**/tests/*'], message: 'Do not import from the tests folder' }],
       },
     ],
+    'prefer-const': [
+      'error',
+      {
+        destructuring: 'any',
+        ignoreReadBeforeAssign: false,
+      },
```

Comment:
I see this is the default configuration for `prefer-const`. I used to omit the default configuration. Do you prefer to keep it explicitly?